packages/xchain-thorchain/src/client.ts

```diff
@@ -201,45 +206,107 @@ class Client implements ThorchainClient, XChainClient {
     }
   }
 
-  deposit = async ({ asset, amount, recipient, memo }: TxParams): Promise<TxHash> => {
-    return this.transfer({ asset, amount, recipient, memo })
+  private getDefaultStdTxFee = (): StdTxFee => {
+    return {
+      amount: [],
+      gas: '10000000',
+    }
+  }
```

Comment:
Why do we need to have another "default fee" function? Note: A frontend needs to know which are the default fees, but this function is private ...